chain-impl-mockchain/src/txbuilder.rs

```diff
@@ -0,0 +1,192 @@
+use crate::fee::FeeAlgorithm;
+use crate::key::SpendingSecretKey;
+use crate::transaction as tx;
+use crate::value::{Value, ValueError};
+use chain_addr::Address;
+
+/// Possible error for the builder.
+#[derive(Debug, Clone)]
+pub enum Error {
+    TxInvalidNoInput,
+    TxInvalidNoOutput,
```

Comment:
These 2 errors are not used anywhere, I hope that's ok? Should they be cleaned up?

Reply:
will use them now :) thanks